libs/common-utilities/src/constants/endpoints.ts

```diff
@@ -5,6 +5,22 @@ export const ENDPOINTS: KeychainList = {
     '0x4': 'https://api.thegraph.com/subgraphs/name/hausdao/daohaus-v3-rinkeby',
     '0x2a': 'https://api.thegraph.com/subgraphs/name/hausdao/daohaus-v3-kovan',
   },
+  // RPC: {
```

Comment:
Commenting out until process errors are fixed. Loacing RPCs in Connect package currently

Reply:
Let's create a new ticket to build this out as an input. 

Users of the package will need to pass the rpc.

Reply:
I am leaning towards hardcoding these